<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextTax_ID">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tax_ID</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextPIN">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>104</width>
      <height>70</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>PIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButtonSubmit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReset">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginSubmitResultLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
